pkg/volumes/openstack/volumes.go

```diff
@@ -352,13 +354,58 @@ func GetDevicePath(volumeID string) (string, error) {
 	return "", fmt.Errorf("Failed to find device for the volumeID: %q", volumeID)
 }
 
+// probeVolume probes volume in compute
+// see issue https://github.com/kubernetes/cloud-provider-openstack/issues/705
+func probeVolume() error {
+	// rescan scsi bus
+	scsiPath := "/sys/class/scsi_host/"
+	if dirs, err := ioutil.ReadDir(scsiPath); err == nil {
```

Comment:
Some warning on errors from ReadDir or WriteFile would be good here, so that we know if we aren't able to trigger a probe (e.g. because we're in a container).  Maybe also a V(2).Infof for each /scan path we write, so that we affirmatively know it's working also (at v=2)